sqlservice/client.py

```diff
@@ -119,9 +123,13 @@ def __init__(self,
             'SQL_MAX_OVERFLOW': None,
             'SQL_AUTOCOMMIT': False,
             'SQL_AUTOFLUSH': True,
-            'SQL_EXPIRE_ON_COMMIT': True
+            'SQL_EXPIRE_ON_COMMIT': True,
         }
 
+        # pool_pre_ping was added in SQLAlchemy 1.2.0
+        if sa.__version__ >= '1.2.0':
+            self.config['SQL_POOL_PRE_PING'] = False
```

Comment:
Actually, you could just make this `None` and then be able to include it in the `self.config = {...}` section. Config values that are `None` won't be included when creating the engine options. That way we don't need to check the SQLAlchemy version.